dist_tables/append_distribution.rst

```diff
@@ -55,38 +55,41 @@ This function informs Citus that the github_events table should be distributed b
+The `\\copy <http://www.postgresql.org/docs/current/static/app-psql.html#APP-PSQL-META-COMMANDS-COPY>`_
+command is used to copy data from a file to a distributed table while handling
+replication and failures automatically. You can also use the server side `COPY command <http://www.postgresql.org/docs/current/static/sql-copy.html>`_. 
+However, in general, \\copy is preferable over COPY in examples, since the latter requires superuser, whereas \\copy does not. \\copy actually sends a COPY .. FROM STDIN to the server.
 
-Note: You can use the psql client if you do not wish to use \stage to ingest data. In upcoming releases, \stage will be replaced by more native ingest methods which will remove the need for csql.
-
-The \stage command borrows its syntax from the `client-side \copy command <http://www.postgresql.org/docs/9.5/static/app-psql.html>`_ in PostgreSQL. Behind the scenes, \stage first opens a connection to the master and fetches candidate workers on which to create new shards. Then, the command connects to these workers, creates at least one shard there, and uploads the data to the shards. The command then replicates these shards on other workers until the replication factor is satisfied and fetches statistics for these shards. Finally, the command stores the shard metadata with the master.
+Behind the scenes, \\copy first opens a connection to the master using the provided master_host option and fetches candidate workers on which to create new shards. Then, the command connects to these workers, creates at least one shard there, and uploads the data to the shards. The command then replicates these shards on other workers until the replication factor is satisfied and fetches statistics for these shards. Finally, the command stores the shard metadata with the master.
 
 ::
 
     SET citus.shard_max_size TO '64MB';
-    SET citus.shard_replication_factor TO 1;
-    \stage github_events from 'github_events-2015-01-01-0.csv' WITH CSV;
+    \copy github_events from 'github_events-2015-01-01-0.csv' WITH (format CSV, master_host 'master-host-101');
 
 Citus assigns a unique shard id to each new shard and all its replicas have the same shard id. Each shard is represented on the worker node as a regular PostgreSQL table with name 'tablename_shardid' where tablename is the name of the distributed table and shardid is the unique id assigned to that shard. One can connect to the worker postgres instances to view or run commands on individual shards.
 
-By default, the \stage command depends on two configuration parameters for its behavior. These are called citus.shard_max_size and citus.shard_replication_factor.
+By default, the \\copy command depends on two configuration parameters for its behavior. These are called citus.shard_max_size and citus.shard_replication_factor.
```

Comment:
There's some inconsistency w.r.t the number of \'s in this file. Is two the right amount?